src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAsyncCollector.cs

```diff
@@ -1,50 +1,93 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.ServiceBus.Messaging;
 
 namespace Microsoft.Azure.WebJobs.ServiceBus
 {
-    // Core object to send events to EventHub. 
-    // Any user parameter that sends events will eventually get bound to this object. 
-    // This gets wrappers with various adapters and passed to the user function. 
-    internal class EventHubAsyncCollector : IAsyncCollector<EventData>
+    /// <summary>
+    /// Core object to send events to EventHub. 
+    /// Any user parameter that sends EventHub events will eventually get bound to this object. 
+    /// This will queue events and send in batches, also keeping under the 256kb event hub limit per batch. 
+    /// </summary>
+    public class EventHubAsyncCollector : IAsyncCollector<EventData>
```

Comment:
I just pushed public surface tests for ServiceBus and your new Logging. Please pull and rebase. You should also review the public types in the Logging assembly, now that this test exposes the list. See commit https://github.com/Azure/azure-webjobs-sdk/commit/af8406f9faeebbd5649fa9038ace3c257cf41227